number.csv

```diff
@@ -0,0 +1 @@
+Preston 8
```

Comment:
@phess2 @sfelshin This does not look good to me.  Instead of adding to the existing file `numbers.csv` you created a new file named `number.csv` (note the lack of `s`).  This is an example of a situation where automated checks are not great, and this is why we do code review.  To @phess2 , I suggest clicking "Revert" to open a new PR that performs the opposite changes of this one, and assign that for review again.  Then, take up adding your favorite number in a new PR after that.